content/200-orm/100-prisma-schema/10-overview/04-location.mdx

```diff
@@ -111,35 +103,14 @@ For example, assuming `schema.prisma` defines the `datasource`, here's how how n
 └── schema.prisma
 ```
 
-
-### How to use existing Prisma CLI commands with multiple Prisma schema files
-
-For most Prisma CLI commands, no changes will be necessary to work with a multi-file Prisma schema. Only in the specific cases where you need to supply a schema via an option will a command need to be changed. In these cases, simply replace references to a file with a directory. As an example, the following `prisma db push` command:
-
-```terminal
-npx prisma db push --schema custom/path/to/my/schema.prisma
-```
-
-becomes the following:
-
-```terminal
-npx prisma db push --schema custom/path/to/my/schema  # note this is now a directory!
-```
-
 ### Tips for multi-file Prisma Schema
 
-We’ve found that a few patterns work well with this feature and will help you get the most out of it:
-
-- Organize your files by domain: group related models into the same file. For example, keep all user-related models in `user.prisma` while post-related models go in `post.prisma`. Try to avoid having “kitchen sink” schema files.
+We've found that a few patterns work well with this feature and will help you get the most out of it:
 
+- Organize your files by domain: group related models into the same file. For example, keep all user-related models in `user.prisma` while post-related models go in `post.prisma`. Try to avoid having "kitchen sink" schema files.
```

Comment:
I didn't write this but I assume it means something like a "randomly mixed group of things". I think we can just drop it though, don't think it's needed.

```suggestion
- Organize your files by domain: group related models into the same file. For example, keep all user-related models in `user.prisma` while post-related models go in `post.prisma`. 
```